etna/collections/models.py

```diff
@@ -543,26 +542,26 @@ class Highlight(Orderable):
         on_delete=models.SET_NULL,
         verbose_name=_("image"),
     )
-    long_description = RichTextField(
-        verbose_name=_("long description"),
-        features=settings.RESTRICTED_RICH_TEXT_FEATURES,
-        max_length=400,
-    )
 
     panels = [
         FieldPanel("image"),
-        FieldPanel("long_description"),
     ]
 
     def clean(self) -> None:
-        if self.image and self.image.record is None:
-            raise ValidationError(
-                {
-                    "image": [
-                        "Only images with a 'record' specified can be used for highlights."
-                    ]
-                }
-            )
+        if self.image:
+            error_fields = []
+            if self.image.record is None:
+                error_fields.append("'record'")
+            if self.image.description == "":
+                error_fields.append("'description'")
+            if error_fields:
+                raise ValidationError(
+                    {
+                        "image": [
+                            f"Only images with a {' and '.join(error_fields)} specified can be used for highlights."
```

Comment:
Hey @jamesbiggs,

I can see what you're doing here and why you're doing it. It's smart, but it doesn't have to be (although this would be very applicable to other parts of the project going forward). As _both_ variables are requied to be used as a highlight it would be more user friendly to display the full conditions i.e. "Only images with a record and description can be used as a highlight".

Reply:
I would think something along the lines of 

```
if self.image:
    if not self.image.record or not self.image.description:
         # Raise an error 

```

Just a small prod, you can you use a python interpreter to quickly check your logic when doing this type of stuff; 

Python 3.10.6 (main, Mar 10 2023, 10:55:28) [GCC 11.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> string = ""
>>> bool(string)
False

tldr; "" is evaluated as `False` while any character is evaluated as `True`